src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -125,16 +126,46 @@ public ShooterSubsystem(final ShooterHoodSubsystem shooterHood, ShuffleboardTab
 
     NetworkTableEntry shooterPIDchooser = tab.add("Shooter PID", true).withWidget(BuiltInWidgets.kToggleSwitch)
         .getEntry();
+
     shooterPIDchooser.addListener(notice -> enablePID = notice.value.getBoolean(),
         EntryListenerFlags.kNew | EntryListenerFlags.kUpdate);
+
+    // Configures a Shuffleboard widget for flywheel speed. It will display as a
+    // graph of the flywheel speed from the last 20 seconds.
+    flywheelSpeed = tab.add("Flywheel Speed (RPM)", 0).withWidget(BuiltInWidgets.kNumberBar).getEntry();
+
+    // Adds a Shuffleboard widget to show whether the flywheel is spinning within a
+    // certain tolerance of the setpoint. See isFlywheelReady().
+    isFlywheelReady = tab.add("Is Flywheel Ready?", false).withWidget(BuiltInWidgets.kBooleanBox).getEntry();
+
+    shooterSetpoint = tab.addPersistent("Shooter Setpoint", 4500).withWidget(BuiltInWidgets.kTextView).getEntry();
+    shooterSetpoint.addListener(notice -> setpoint = notice.value.getDouble(),
+        EntryListenerFlags.kNew | EntryListenerFlags.kUpdate);
+
+    kPSlider = tab.addPersistent("Shooter kP", 5e-5).withWidget(BuiltInWidgets.kTextView).getEntry();
```

Comment:
There should be no hardcoded constants in these calls.  Especially since these are tuning values and already exist elsewhere.  Keeping them in sync is high maintenance.